src/tsf/precomp.h

```diff
@@ -23,7 +23,7 @@ Revision History:
 #include <windows.h>
 #include <ole2.h>
 
-#include <atlbase.h>        // ATL base
+#include <wil/com.h>        // COM smart pointer

```

Comment:
Any reason to not just send this into `LibraryIncludes.h` which is already included in `tsf/precomp.h`?

That's where we're pulling in pretty much all the other WIL stuff.